modules/monitoring-granting-users-permission-to-configure-monitoring-for-user-defined-projects.adoc

```diff
@@ -6,7 +6,7 @@
 [id="granting-users-permission-to-configure-monitoring-for-user-defined-projects_{context}"]
 = Granting users permission to configure monitoring for user-defined projects
 
-You can grant users permission to configure monitoring for user-defined projects.
+As a cluster administrator, you can assign the `user-workload-monitoring-config-edit` role to a user to grant permission to configure and manage monitoring for user-defined projects without giving them permission to configure and manage core {product-title} monitoring components.
```

Comment:
```suggestion
As a cluster administrator, you can assign the `user-workload-monitoring-config-edit` role to a user to grant permission to configure and manage monitoring for user-defined projects without giving them permission to configure and manage core {product-title} monitoring components. User with `user-workload-monitoring-config-edit` rolebinding have full permission for  `user-workload-monitoring-config` configmap under `openshift-user-workload-monitoring` project
```
based on the `user-workload-monitoring-config-edit` role, @machine424 do you think the suggestion is fine?
```
$ oc -n openshift-user-workload-monitoring get role user-workload-monitoring-config-edit -oyaml
apiVersion: rbac.authorization.k8s.io/v1
kind: Role
metadata:
  creationTimestamp: "2024-03-21T00:43:51Z"
  labels:
    app.kubernetes.io/managed-by: cluster-monitoring-operator
    app.kubernetes.io/part-of: openshift-monitoring
  name: user-workload-monitoring-config-edit
  namespace: openshift-user-workload-monitoring
  resourceVersion: "18856"
  uid: 678912d7-7a4f-4f27-a69b-7952c9e268a4
rules:
- apiGroups:
  - ""
  resourceNames:
  - user-workload-monitoring-config
  resources:
  - configmaps
  verbs:
  - get
  - list
  - watch
  - patch
  - update
```

Reply:
Hello @juzhao we already mention this information [here in the role descriptions](https://73398--ocpdocs-pr.netlify.app/openshift-enterprise/latest/monitoring/enabling-monitoring-for-user-defined-projects#granting-users-permission-to-monitor-user-defined-projects_enabling-monitoring-for-user-defined-projects). Do we still want to repeat this information here as well?

Reply:
@eromanova97 ok, then no need now
/lgtm